lib/SetupChecks/CirclesAppIsEnableCheck.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCA\Collectives\SetupChecks;
+
+use OCP\App\IAppManager;
+use OCP\IL10N;
+use OCP\SetupCheck\ISetupCheck;
+use OCP\SetupCheck\SetupResult;
+
+class CirclesAppIsEnableCheck implements ISetupCheck {
+	public function __construct(
+		private IL10N       $l10n,
+		private IAppManager $appManager
+	) {
+	}
+
+
+	public function getCategory(): string {
+		return 'app';
+	}
+
+	public function getName(): string {
+		return $this->l10n->t('Circles App Enabled');
+	}
+
+	public function run(): SetupResult {
+		if ($this->appManager->isEnabledForUser('circles')) {
+			return SetupResult::success($this->l10n->t('Circles App is enabled'));
+		}
+
+		return SetupResult::warning($this->l10n->t('Collectives app detected, but Circles app is not enabled. Please enable Circles app for enhanced collaboration with your team'));
```

Comment:
```suggestion
		return SetupResult::error($this->l10n->t('The teams app is not enabled, but is required for Collectives to work.'));
```

This is actually a hard requirement so we should make this more clear.